src/client/sandbox/node/window.js

```diff
@@ -525,6 +532,30 @@ export default class WindowSandbox extends SandboxBase {
         nativeMethods.objectDefineProperty
             .call(window.Object, window.MessageEvent.prototype, 'origin', originPropDescriptor);
 
+        if (window.PerformanceEntry) {
+            if (nativeMethods.performanceEntryNameGetter) {
+                const namePropDescriptor = nativeMethods.objectGetOwnPropertyDescriptor
```

Comment:
For overriding the property descriptor use the `overrideDescriptor` function - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/override-descriptor.js